barretenberg/acir_tests/flows/honk_sol.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -eu
+
+export PROOF="$(pwd)/proof"
+export PROOF_AS_FIELDS="$(pwd)/proof_fields.json"
+
+# Create a proof, write the solidity contract, write the proof as fields in order to extract the public inputs
+$BIN prove_keccak_ultra_honk -o proof
```

Comment:
I have included a second bb command that uses the UltraKeccak flavor for proving, in its current form this is not very user friendly.